src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade-form/upgrade-start-modal.component.ts

```diff
@@ -44,6 +44,10 @@ export class UpgradeStartModalComponent implements OnInit {
       useImage: new FormControl(false),
       customImageName: new FormControl(null)
     });
+    if (this.versions === undefined) {
+      const availableVersionsControl = this.upgradeForm.get('availableVersions');
+      availableVersionsControl.clearValidators();
```

Comment:
are we sure the required validator works here if we put the field empty. Because we are not adding it in the `FormControl` above. Maybe explicility adding the validator for customImageName when version is null would be good!
```suggestion
      availableVersionsControl.clearValidators();
      
      const customImageNameControl = this.upgradeForm.get('customImageName');
      customImageNameControl.setValidators(Validators.required);
      customImageNameControl.updateValueAndValidity();
```

Reply:
I thought the validator for `customImageName` would still be there as its a diff. control

Reply:
Btw I did notice required validator was missing for custom image.